content/blog/2021-02-18-dvc-2-0-pre-release.md

```diff
@@ -0,0 +1,282 @@
+previous DVC version 1.0. See
+[params option](https://dvc.org/doc/command-reference/run#for-displaying-and-comparing-data-science-experiments)
+in `dvc run` for more details.
+
+### Iterating over params with foreach stages
+
+Iterating over params was a frequently requested feature. Now users can define
+multiple similar stages by a single command.
+
+```yaml
+stages:
+  build:
+    foreach:
+      gb:
+        thresh: 15
+        filename: 'model-gb.hdf5'
+      us:
+        thresh: 10
+        filename: 'model-us.hdf5'
+    do:
+      cmd: >-
+        python script.py --out ${item.filename} --thresh ${item.thresh}
+      outs:
+        - ${item.filename}
+```
+
+## Lightweight ML experiments
+
+DVC uses Git as a foundation for ML experiments. This solid foundation makes
+each ML experiment reproducible and accessible from Git history. This Git-based
+approach works very well for ML projects with mature ML models when only a few
+new experiments per day are running. However, in more active development when
+dozens or hundreds of experiments need to be run in a single day, Git creates
+overhead - each experiment run requires additional Git commands
+`git add/commit`, and comparing all experiments is difficult.
+
+We introduce lightweight experiments in DVC 2.0! This is the way of
+auto-tracking without any overhead from ML engineers:
+
+```yaml
+$ dvc exp run
+...
+Reproduced experiment(s): exp-0c0f7
+…
+$ dvc exp run --params learning_rate=0.03
+Reproduced experiment(s): exp-bc5cd
+
+
+$ dvc exp run --params learning_rate=0.025
+Reproduced experiment(s): exp-49181
+
+$ vi train.py  # edit code
+$ dvc exp run --params learning_rate=0.025
+Reproduced experiment(s): exp-a814a
+```
+
+See all the runs:
+
+```yaml
+$ dvc exp show --no-pager
+```
+
+Any of the runs can be easily obtained with code, data and models:
+
+```yaml
+$ dvc apply exp-49181 Changes for experiment 'exp-0c0f7' have been applied to
+your current workspace.
+```
+
+## Model checkpoints tracking
+
+ML model checkpoints are an essential part of deep learning. ML engineers prefer
+to save the model files (or weights) at checkpoints during a training process
+and return back when metrics start diverging or learning is not fast enough.
+
+The checkpoints create a different dynamic around ML modeling process and need a
+special support from the toolset:
+
+1. Track and save model checkpoints (DVC outputs) periodically, not only the
+   final result or training epoch.
+2. Save metrics corresponding to each of the checkpoints.
+3. Reuse checkpoints - warm-start training with an existing model file,
+   corresponding code, dataset version and metrics.
+
+This new behaviour is supported in DVC 2.0. Now, DVC can version all your
+checkpoints with corresponding code and data. It brings reproducibility of DL
+processes to the next level - every checkpoint is reproducible.
+
+This is how you define checkpoints with live-metrics:
+
+```
```

Comment:
should be empty?

Reply:
Probably WIP (I'm waiting for that to review/copy edit the 2nd half).